autospec/build.py

```diff
@@ -158,6 +158,28 @@ def failed_pattern(line, pattern, verbose, buildtool=None):
             warned_about.add(s)
 
 
+def parse_buildroot_log(filename, returncode):
+    """Handle buildroot log contents."""
+    if returncode == 0:
+        return
+    else:
```

Comment:
May as well save the indent and skip the else clause here since you return otherwise.

Reply:
Okay, will fix shortly, thanks